src/app/app.module.ts

```diff
@@ -46,7 +47,8 @@ const matomoTrackers = environment.matomoSiteId ? [
       trackers: matomoTrackers,
       routeTracking: {
         enable: true,
-      }
+      },
+        requireConsent: flags.cookieBannerEnabled,
```

Comment:
This means we won't send tracking data to Matamo at all without user consent. There is also an option to send tracking data without setting cookies, but I think we want to avoid both.